src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day--sun calendar__month-length--29">
```

Comment:
Names are not written with BEM rules.
These are modificators, not elements, so
calendar__start-day--sun should be like calendar--start-day--sun
calendar__month-length--29 should be like calendar--month-length--29

Reply:
Thank you for your comments, i see the problem, and try to fixed it. Because, it really worked, but not now. You can see screen screenshot... I get confused...
<img width="515" alt="Screenshot 2023-03-03 at 07 44 04" src="https://user-images.githubusercontent.com/122730083/222651201-78c01645-0673-49c5-9ee7-b1480fa9cec1.png">